File: Java/src/com/sonicscholar/tictactoe/Coordinate.java
package com.sonicscholar.tictactoe;

/**
 * This is a class to keep track of a row and column
 * for a board space
 */
public class Coordinate extends Tuple<Integer, Integer>{

    public int getRow() {
        return this.value1;
    }
    public int getColumn() {
        return this.value2;
    }
    
    public Coordinate(int row, int column) {
        super(row, column);
    }

}

File: C_Sharp/SonicScholar.SimpleBoardGame.Framework/TicTacToe/Coordinate.cs
using System;

namespace TicTacToe
{
    /// <summary>
    /// This is a class to keep track of a row and column
    /// for a board space
    /// </summary>
    public class Coordinate : Tuple<int, int>
    {

        public int Row => this.Item1;
        public int Column => this.Item2;

        public Coordinate(int row, int column)
        : base(row, column)
        {
        }

    }
}